org-formation/060-cloudtrail/_tasks.yaml

```diff
@@ -37,6 +37,7 @@ CloudTrail:
     KmsKeyAdminPrincipalArns:
       - !Sub 'arn:aws:iam::${AWS::AccountId}:root'
       - !Sub 'arn:aws:iam::${AWS::AccountId}:role/OrganizationAccountAccessRole'
+      - 'arn:aws:iam::531805629419:role/github-oidc-sage-bionetwo-ProviderRoleorganization-1EY1CRUF48VP5'
```

Comment:
Why does GitHub need to access KMS?  E.g., does it transfer secrets from its own store to KMS?

Reply:
@brucehoff because that is the role that we setup for github actions.  it needs access to the key to make changes.

Reply:
@zaro0508 I see.  I don't see any other references to this role, in this PR or in the https://github.com/Sage-Bionetworks-IT/organizations-infra repo'.  Does GitHub Actions not need access to other AWS services besides KMS?  (Sorry if this is a dumb question.)

Reply:
@brucehoff the role is defined here: https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/650-identity-providers/_tasks.yaml 

The role has admin access, however admin access does not automatically give accesss to KMS keys.  To provide KMS access we need to explicitly set every principal.  That's how it works in AWS.